package proto_ff;

import "yd_fieldoptions.proto";


message E_ShenequipEquip
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "装备ID"];
	optional string m_name = 2[(yd_fieldoptions.field_cname) = "装备名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "装备大类"];
	optional int32 m_grade = 4[(yd_fieldoptions.field_cname) = "装备评级"];
	optional int32 m_subtype = 5[(yd_fieldoptions.field_cname) = "装备子类"];
	optional int32 m_level = 6[(yd_fieldoptions.field_cname) = "穿戴等级限制"];
	optional int32 m_quality = 7[(yd_fieldoptions.field_cname) = "装备品质"];
	optional int32 m_star = 8[(yd_fieldoptions.field_cname) = "初始星级"];
	optional int32 m_attributeid = 9[(yd_fieldoptions.field_cname) = "属性ID"];
	optional string m_decompose = 10[(yd_fieldoptions.field_cname) = "分解获得", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_iscanbind = 11[(yd_fieldoptions.field_cname) = "是否绑定"];
	optional int32 m_istradebind = 12[(yd_fieldoptions.field_cname) = "是否交易绑定"];
	optional int32 m_time = 13[(yd_fieldoptions.field_cname) = "限时装备"];
	optional int32 m_broadcast = 14[(yd_fieldoptions.field_cname) = "是否播放跑马灯"];
	optional int32 m_exclusiveid = 15[(yd_fieldoptions.field_cname) = "专属技能"];
	optional string m_secretskill = 16[(yd_fieldoptions.field_cname) = "秘技技能", (yd_fieldoptions.field_bufsize) = 60];
	optional string m_crypticskill = 17[(yd_fieldoptions.field_cname) = "暗术技能", (yd_fieldoptions.field_bufsize) = 60];
	optional string m_passiveskill = 18[(yd_fieldoptions.field_cname) = "被动技能", (yd_fieldoptions.field_bufsize) = 80];
}


message Sheet_ShenequipEquip
{
	repeated E_ShenequipEquip E_ShenequipEquip_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_ShenequipAttAttributeDesc
{
	optional int32 m_type = 1[(yd_fieldoptions.field_cname) = ""];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "数值"];
}

message E_ShenequipAttStarDesc
{
	optional int32 m_type = 1[(yd_fieldoptions.field_cname) = ""];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "数值"];
}

message E_ShenequipAtt
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "属性ID"];
	repeated E_ShenequipAttAttributeDesc m_attribute = 2[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 4];
	repeated E_ShenequipAttStarDesc m_star = 3[(yd_fieldoptions.field_cname) = "星级属性", (yd_fieldoptions.field_arysize) = 7];
}


message Sheet_ShenequipAtt
{
	repeated E_ShenequipAtt E_ShenequipAtt_List = 1[(yd_fieldoptions.field_arysize)=11000];
}

message E_ShenequipLvup
{
	optional int32 m_lv = 1[(yd_fieldoptions.field_cname) = "注灵等级"];
	optional int32 m_itemid = 2[(yd_fieldoptions.field_cname) = "升级道具"];
	optional int32 m_num = 3[(yd_fieldoptions.field_cname) = "升级数量"];
	optional int32 m_advanceitem = 4[(yd_fieldoptions.field_cname) = "进阶道具"];
	optional int32 m_advancenum = 5[(yd_fieldoptions.field_cname) = "进阶数量"];
	optional int32 m_attributeid = 6[(yd_fieldoptions.field_cname) = "升级属性ID"];
	optional int32 m_stepid = 7[(yd_fieldoptions.field_cname) = "进阶属性ID"];
}


message Sheet_ShenequipLvup
{
	repeated E_ShenequipLvup E_ShenequipLvup_List = 1[(yd_fieldoptions.field_arysize)=40];
}

message E_ShenequipClear
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "装备大类"];
	optional int32 m_lv = 3[(yd_fieldoptions.field_cname) = "等级"];
	optional int32 m_itemid = 4[(yd_fieldoptions.field_cname) = "铭刻道具"];
	optional int32 m_num = 5[(yd_fieldoptions.field_cname) = "消耗数量"];
	optional int32 m_breakitem = 6[(yd_fieldoptions.field_cname) = "突破道具"];
	optional int32 m_breaknum = 7[(yd_fieldoptions.field_cname) = "突破数量"];
	optional int32 m_breakprogress = 8[(yd_fieldoptions.field_cname) = "突破进度值"];
	optional int32 m_randomcounts = 9[(yd_fieldoptions.field_cname) = "随机条目数"];
	optional int32 m_minus = 10[(yd_fieldoptions.field_cname) = "负数百分比"];
	optional int32 m_nextid = 11[(yd_fieldoptions.field_cname) = "下段ID"];
	repeated int32 m_percent = 12[(yd_fieldoptions.field_cname) = "百分比区间值", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_ShenequipClear
{
	repeated E_ShenequipClear E_ShenequipClear_List = 1[(yd_fieldoptions.field_arysize)=140];
}

message E_ShenequipAwakenAttributeDesc
{
	optional int32 m_type = 1[(yd_fieldoptions.field_cname) = ""];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "数值"];
}

message E_ShenequipAwaken
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "装备大类"];
	optional int32 m_lv = 3[(yd_fieldoptions.field_cname) = "技能等级"];
	optional int32 m_itemid = 4[(yd_fieldoptions.field_cname) = "觉醒材料"];
	optional int32 m_num = 5[(yd_fieldoptions.field_cname) = "材料数量"];
	optional int32 m_awakeskillid = 6[(yd_fieldoptions.field_cname) = "觉醒技能"];
	repeated E_ShenequipAwakenAttributeDesc m_attribute = 7[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_ShenequipAwaken
{
	repeated E_ShenequipAwaken E_ShenequipAwaken_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_ShenequipStarupMaterialDesc
{
	optional int32 m_grade = 1[(yd_fieldoptions.field_cname) = "评级"];
	optional int32 m_quality = 2[(yd_fieldoptions.field_cname) = "品质"];
	optional int32 m_num = 3[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 m_type = 4[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 m_star = 5[(yd_fieldoptions.field_cname) = "星级"];
}

message E_ShenequipStarup
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_grade = 2[(yd_fieldoptions.field_cname) = "装备评级"];
	optional int32 m_quality = 3[(yd_fieldoptions.field_cname) = "装备品质"];
	optional int32 m_stars = 4[(yd_fieldoptions.field_cname) = "装备星级"];
	optional int32 m_nextid = 5[(yd_fieldoptions.field_cname) = "下级装备"];
	optional int32 m_lastid = 6[(yd_fieldoptions.field_cname) = "上级装备"];
	optional int32 m_percent = 7[(yd_fieldoptions.field_cname) = "提升增加的百分比"];
	repeated E_ShenequipStarupMaterialDesc m_material = 8[(yd_fieldoptions.field_cname) = "升级材料", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_ShenequipStarup
{
	repeated E_ShenequipStarup E_ShenequipStarup_List = 1[(yd_fieldoptions.field_arysize)=80];
}

message E_ShenequipPreposeRewardDesc
{
	optional int32 m_item = 1[(yd_fieldoptions.field_cname) = "道具"];
	optional int32 m_num = 2[(yd_fieldoptions.field_cname) = "数量"];
}

message E_ShenequipPrepose
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_condition = 2[(yd_fieldoptions.field_cname) = "条件"];
	optional int32 m_value = 3[(yd_fieldoptions.field_cname) = "条件值"];
	repeated E_ShenequipPreposeRewardDesc m_reward = 4[(yd_fieldoptions.field_cname) = "奖励", (yd_fieldoptions.field_arysize) = 5];
}


message Sheet_ShenequipPrepose
{
	repeated E_ShenequipPrepose E_ShenequipPrepose_List = 1[(yd_fieldoptions.field_arysize)=20];
}
